packages/gluestick/shared/lib/getHttpClient.js

```diff
@@ -122,3 +130,13 @@ export default function getHttpClient(
 
   return client;
 }
+
+function hostMatch(config) {
+  if (!config || !config.baseURL || !config.url) {
+    return false;
+  }
+
+  const originalHostName = new URL(config.baseURL).hostname;
```

Comment:
Please use `parse` function from `url` module, since there's no `URL` constructor in node 6. This is also the reason why CI is failing.